.github/CODEOWNERS

```
@@ -14,6 +12,7 @@
 /projects/miopen/ @BrianHarrisonAMD @BradPepersAMD @adickin-amd @JonathanLichtnerAMD
 /projects/composablekernel/ @illsilin @carlushuang @qianfengz @aosewski @poyenc @geyyer @bartekxk @andriy-ca @afagaj @asleepzzz @tenpercent @ThomasNing @coderfeli
 /projects/hipblas-common/ @bnemanich @daineAMD @TorreZuk
+/projects/hipblas-common/**/CMakeLists.txt @ROCm/rocm-math-lib-build-infra
```

should we just add codeowners for this
```suggestion
/projects/hipblas-common/** @ROCm/rocm-math-lib-build-infra
``` 
?

just curious. if it's only for CmakeLists.txt, then good with me

---

Correct, we only want to be codeowners for the CMakeLists.txt files

---

just making sure :)